### DISCUSSION TITLE
Flux processing

### DISCUSSION THREAD

**Discussion by metazool** [ukceh-rse/community#19] [General]
Placeholder discussion! For the topic of flux processing, raised by @mattfry-ceh in the recent Environmental Digital Strategy Steering Group that we [gave a quick presentation to](https://github.com/ukceh-rse/digital-strategy-2024-pres)

Flux processing work is being done in the FDRI project that should have applicability to other areas. How can RSEs help (others to) ensure that this work is discovered and reused, not just FAIR in principle?

**Comment by mattfry-ceh** [comment]
Thanks Jo. @holcoo should be involved in this as she's aware of FDRI flux needs but also processing approaches in COSMOS and the UKCEH flux network, and the Edinburgh group. It sounds like there is a need for a general reusable flux processing workflow for a number of projects (each with slightly different requirements) that could implement best practice and deliver consistent and interoperable data.

**Comment by holcoo** [comment]
Resources for future consideration:

EC data processing software available: https://fluxnet.org/2017/10/10/toolbox-a-rolling-list-of-softwarepackages-for-flux-related-data-processing/

Data formats (perhaps separate process when completing e.g. annual publications) for submitting to FLUXNET via the European Fluxes Database: https://www.europe-fluxdata.eu/home/guidelines/how-to-submit-data/general-information

Potential auxiliary data (AI plant imagery) to consider: https://floraincognita.com/floraflux/

**Comment by jmarshrossney** [comment]
Hi @holcoo. Thanks for the links!

I'm really curious to know if there's been any work on this in FDRI.

I'd like to use some eddy covariance data for calibrating a land surface model, so I'd be very happy to be a 'beta tester' for any pipelines under development!

**Reply by holcoo** [reply]
Hi @jmarshrossney. We're looking into this based on existing methods used across UKCEH (e.g. differences mentioned in this (outdated) page: https://uk-scape.ceh.ac.uk/our-science/projects/UK-GHG-Flux-Network). FDRI will have EC sites, primarily for water fluxes.

If you're after existing EC observations, have you spoken to the Wallingford flux team (e.g. Brenda D'Acunha / Alex Cumming) or seen the EIDC catalogue: https://catalogue.ceh.ac.uk/documents/578b8f2d-4cf5-4d4c-ad7a-281c43f27808? Feel free to message me separately as well if you have any questions.

Great to have your input as we investigate FDRI EC data workflows and outputs!

**Reply by jmarshrossney** [reply]
@vmyrgiotis
